controllers/error.controller.js

```diff
@@ -1,3 +1,6 @@
 exports.errorHandler = (err, req, res, next) => {
```

Comment:
This is a great function for your custom errors (e.g. Promise.reject), but you should also look at creating one for PSQL errors.